projects/packages/my-jetpack/_inc/hooks/use-connection-watcher/index.js

```diff
@@ -1,5 +1,6 @@
 import { __, sprintf } from '@wordpress/i18n';
```

Comment:
It looks like this hook and `use-route-blocker` were not converted to TypeScript. Was this intended?

Reply:
Ah, yes thanks for spotting!

Actually `use-route-blocker` is not used and I forgot to delete it. I rewrited `use-connection-watcher` now, and actually discovered some type problems, so improved `useAllProducts()` hook - check out if that makes sense.